content/en/docs/getting-started/_index.md

```diff
@@ -60,73 +60,140 @@ olm-operator       1/1     1            1           5m52s
+```shell
+make run-console-local
 ```
 
-Then create a subscription for the etcd operator:
+Then visit `http://localhost:9000` to view the console.
+
+## Customizing OLM installations 
+
+Deployments of OLM can be stamped out with different configurations by writing a `values.yaml` file and running commands to generate resources.
+
+Here's an example `values.yaml`:
+
+```yaml
+# sets the apiversion to use for rbac-resources. Change to `authorization.openshift.io` for openshift
+rbacApiVersion: rbac.authorization.k8s.io
+# namespace is the namespace the operators will _run_
+namespace: olm
+# watchedNamespaces is a comma-separated list of namespaces the operators will _watch_ for OLM resources.
+# Omit to enable OLM in all namespaces
+watchedNamespaces: olm
+# catalog_namespace is the namespace where the catalog operator will look for global catalogs.
+# entries in global catalogs can be resolved in any watched namespace
+catalog_namespace: olm
+# operator_namespace is the namespace where the operator runs
+operator_namespace: operators
+# OLM operator run configuration
+olm:
+  # OLM operator doesn't do any leader election (yet), set to 1
+  replicaCount: 1
+  # The image to run. If not building a local image, use sha256 image references
+  image:
+    ref: quay.io/operator-framework/olm:local
+    pullPolicy: IfNotPresent
+  service:
+    # port for readiness/liveness probes
+    internalPort: 8080
+# catalog operator run configuration
+catalog:
+  # Catalog operator doesn't do any leader election (yet), set to 1
+  replicaCount: 1
+  # The image to run. If not building a local image, use sha256 image references
+  image:
+    ref: quay.io/operator-framework/olm:local
+    pullPolicy: IfNotPresent
+  service:
+    # port for readiness/liveness probes
+    internalPort: 8080
+```
 
-```sh
-$ cat subscription.yaml
+To configure a release of OLM for installation in a cluster:
+
+1. Create a `my-values.yaml` like the example above with the desired configuration or choose an existing one from this repository. The latest production values can be found in [deploy/upstream/values.yaml][deploy-upstream-values].
+
+1. Generate deployment files from the templates and the `my-values.yaml` using `package_release.sh`
+
+   ```bash
+   # first arg must be a semver-compatible version string
+   # second arg is the output directory
+   # third arg is the values.yaml file
+   ./scripts/package_release.sh 1.0.0-myolm ./my-olm-deployment my-values.yaml
+   ```
+
+1. Deploy to kubernetes: `kubectl apply -f ./my-olm-deployment/templates/`
+
+The above steps are automated for official releases with `make ver=0.3.0 release`, which will output new versions of manifests in `deploy/tectonic-alm-operator/manifests/$(ver)`.
+
+## Overriding the Global Catalog Namespace
+
+It is possible to override the Global Catalog Namespace by setting the `GLOBAL_CATALOG_NAMESPACE` environment variable in the catalog operator deployment.
```

Comment:
I'm not sure what this means? We could probably elaborate on this a little more.

Reply:
It is the current info provided by OLM README via the Getting Started section see:

- https://github.com/operator-framework/operator-lifecycle-manager#getting-started
- https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/install/install.md#overriding-the-global-catalog-namespace

Then, if need to be improved I think would better done in a follow up for someone that has  more context than I.